### PR TITLE
Port std-lib to shorter syntax

### LINKED ISSUE #129: Port std-lib to shorter syntax
Derived from https://github.com/rexim/Morganey/pull/122#issuecomment-241257019

### PR DESCRIPTION
**Pull request by rexim**
Close #129

### CODE REVIEW

File: std/boolean.morganey [hunk 1]
```diff
@@ -1,2 +1,2 @@
-true := (\a . (\b . a))
+true := \a.b.a
```
Comment (ForNeVeR) [inline std/boolean.morganey]: Is there a particular reason you're using `\` here and `λ` in `arithmetic.morganey`? If no, then I think it's better to be consistent among all the standard library.

Comment (ForNeVeR) [inline std/boolean.morganey]: Also you haven't used space after last dot in this line, although you're using them in `arighmetic.morganey`.

**Comment by keddelzz** [comment]
I'm not quite sure if that is in the scope of this issue, but I suggest implementing the whole std-lib on a very high semantic level. Therefore I would like to propose to change the definition of

```
false := 0
```

to

```
false := \a.b.b
```

(And the same thing with `nil`)

## 

Otherwise, LGTM :) :+1:

File: std/boolean.morganey (hunk 1 above)
Comment (rexim) [reply std/boolean.morganey]: Done

**Comment by keddelzz** [comment]
This still LGTM after you latest :+1:

**Comment by rexim** [comment]
Thanks everyone!

**Pull request merged**